src/core/model/Stroke.h

```diff
@@ -129,7 +129,7 @@ class Stroke: public AudioElement {
     void setPointVectorInternal(const Range* const snappingBox);
 
 public:
-    void deletePoint(int index);
+    [[maybe_unused]] void deletePoint(int index);
```

Comment:
I think we can remove this function. It make little sense to me to be able to delete any point in a stroke

Reply:
Thanks for the review. I removed that function as well in a second commit.

Merging in 24 hours if no objections are raised.